spring-integration-hazelcast/README.md

```diff
@@ -0,0 +1,301 @@
+SPRING INTEGRATION HAZELCAST SUPPORT
+====================================
+
+## HAZELCAST EVENT-DRIVEN INBOUND CHANNEL ADAPTER 
+
+Hazelcast provides distributed data structures such as 
+
+* com.hazelcast.core.IMap, 
+* com.hazelcast.core.MultiMap, 
+* com.hazelcast.core.IList, 
+* com.hazelcast.core.ISet, 
+* com.hazelcast.core.IQueue, 
+* com.hazelcast.core.ITopic,
+* com.hazelcast.core.ReplicatedMap. 
+
+It also provides event listeners in order to listen to the modifications performed on these data structures. 
+
+* com.hazelcast.core.EntryListener<K, V>
+* com.hazelcast.core.ItemListener<E>
+* com.hazelcast.core.MessageListener<E>
+
+Hazecast Event-Driven Inbound Channel Adapter listens related cache events and sends event messages to defined channel. Its basic definition is as follows : 
+```
+   <int-hazelcast:inbound-channel-adapter channel="mapChannel" 
+					  cache="map" 
+					  cache-events="UPDATED, REMOVED"
+					  cache-listening-policy="SINGLE" /> 
+```
+Basically, Hazecast Event-Driven Inbound Channel Adapter requires following attributes : 
+
+* **channel :** Specifies channel which message is sent. It is mandatory attribute. 
+* **cache :** Specifies the distributed Object reference which is listened. It is mandatory attribute.
+* **cache-events :** Specifies cache events which are listened. It is optional attribute and its default value is ADDED. Its supported values are as follows : 
+
+1. Supported cache event types for IMap and MultiMap : ADDED, REMOVED, UPDATED, EVICTED, EVICT_ALL and CLEAR_ALL. 
+2. Supported cache event types for ReplicatedMap : ADDED, REMOVED, UPDATED, EVICTED. 
+3. Supported cache event types for IList, ISet and IQueue : ADDED, REMOVED. 
+4. There is no need to cache event type definition for ITopic. 
+
+* **cache-listening-policy :** Specifies cache listening policy as SINGLE or ALL. It is optional attribute and its default value is SINGLE. Each Hazelcast inbound channel adapter which listens same cache object with same cache-events attribute, can receive a single event message or all event messages. If this is ALL, all Hazelcast inbound channel adapters which listens same cache object with same cache-events attribute, will receive same event messages. If this is SINGLE, they will receive unique event messages.
+
+Sample namespace and schemaLocation definitions are as follows : 
+```
+xmlns:int-hazelcast= “http://www.springframework.org/schema/integration/hazelcast”
+
+xsi:schemaLocation="http://www.springframework.org/schema/integration/hazelcast
+		  classpath:org/springframework/integration/hazelcast/config/xml/spring-integration-hazelcast-4.2.xsd”
```

Comment:
Why `4.2` ?
This project definitely will be as separate one, like SI-Kafka, SI-Splunk, SI-AWS etc..
So, `1.0`.